build.gradle

```diff
@@ -69,4 +69,8 @@ shadowJar {
     archiveFileName = 'addressbook.jar'
 }
 
+run {
+    enableAssertions = true
```

Comment:
Great idea configuring Gradle to enable assertions instead of using the IntelliJ VM options 👍